src/components/CameraPermissions/Recover/index.tsx

```diff
@@ -71,4 +74,4 @@ const Recover: FunctionComponent<Props> = ({ translate, audio }) => {
   )
 }
 
-export default trackComponent(localised(Recover))
+export default trackComponent(localised(Recover), 'camera_access_denied')
```

Comment:
Same here, used by multiple screens. Need to add the events.

Missing:
`screen_face_face_video_capture_camera_access_denied`
`screen_document_document_video_capture_camera_access_denied`